hyperspy/drawing/utils.py

```diff
@@ -500,7 +507,15 @@ def plot_images(images,
 
     # Get default colormap from pyplot:
     if cmap is None:
-        cmap = plt.get_cmap()
+        cmap = plt.get_cmap().name
+    elif isinstance(cmap, mpl.colors.Colormap):
+        cmap = cmap.name
+    if centre_colormap == "auto":
+        if cmap in MPL_DIVERGING_COLORMAPS:
+            centre_colormap = True
+        else:
+            centre_colormap = False
+        print centre_colormap
```

Comment:
This whole dance with "auto" gets done twice in the code... Any way to combine the two? 

Also `print` here as well